src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -561,6 +561,15 @@ public MaterialStore getMaterialStore()
     }
 
     /**
+     * Called upon completion of an upgrade process.
+     *
+     * @param newLevel The new level
+     */
+    public void onUpgradeComplete(int newLevel) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)